package.json

```diff
@@ -30,6 +30,7 @@
     "build": "babel ./src --out-dir ./dist --copy-files",
     "build:clean": "rimraf ./build",
     "build:watch": "babel ./src --out-dir ./dist --watch",
+    "git:clean": "git checkout -q -- ./dist/*",
```

Comment:
This is fine but we have the `./dist` folder in the `.gitignore` so it shouldn't matter.